### ISSUE TITLE
ITMS-90381: Too many symbol files issue from release build 

### ISSUE THREAD

**Issue by rachelhyman** [mParticle/mparticle-apple-sdk#103]
Hi,

We have the mParticle Apple SDK version 7.15.0 installed via Carthage. When submitting a release build to App Store Connect, we got the following email:
>We identified one or more issues with a recent delivery for your app. Your delivery was successful, but you may wish to correct the following issues in your next delivery:
ITMS-90381: Too many symbol files - These symbols have no corresponding slice in any binary [1A297DE5-7B71-3B6A-9EF8-F6ED393F3D78.symbols].

Using dwarfdump on an archived build (as outlined [here](https://stackoverflow.com/a/38461479)) pointed to the mParticle Apple SDK dSYM as having a matching UUID to the one in the email. A potential solution from the StackOverflow answer might be changing the debug information format for the target from DWARF with dSYM File to just DWARF. It seems that it might be possible for us to do this on our side with Cocoapods, but not with Carthage.

Can you reproduce this issue? Is there a fix?

**Comment by PorterHoskins** [comment]
We are having this same issue.

**Comment by jshier** [comment]
We found that this issue was due to having architectures built into dependencies that the main app doesn't need. I'm not sure what the fix would be for Carthage but for CocoaPods we found that [setting the pod deployment target to iOS 11](https://github.com/CocoaPods/CocoaPods/issues/7111#issuecomment-573709158), which no longer supports the older 32-bit architectures, fixed the warning. Setting the deployment target rather than explicit architectures should be more future proof against new architectures in the future.
